.github/workflows/ci-cd.yml

```diff
@@ -674,46 +673,20 @@ jobs:
           3.10
         - 3.9
         - 3.8
-        - pypy-3.7
-        - 3.7
-        - 3.6
-        - pypy-3.6
         os:
         - ubuntu-22.04
         - ubuntu-20.04
         - macos-11.0
         - macos-latest
         - windows-2019
         - windows-2022
-        exclude:
+        # exclude:
         # NOTE: Windows PyPy jobs are excluded to address the tox bug
```

Comment:
These are comments referring to the removed lines, they need to go too.

Reply:
I retained the comments because they don't refer to specific Python versions but to other factors (OS, PyPy) which seemingly apply to other Python versions and would be relevant should PyPy be added to the matrix.

Reply:
Removed. We can come back here if we need it in the future.